src/test/java/pages/OrderGeneralPage.java

```diff
@@ -0,0 +1,84 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+public class OrderGeneralPage {
+
+   private final WebDriver driver;
+   public OrderGeneralPage(WebDriver driver) {
+       PageFactory.initElements(driver, this);
+       this.driver = driver;
+   }
+
+   @FindBy(xpath = "//button[@aria-label='Add to cart']")
+   List<WebElement> setOfAddToCartBtn;
```

Comment:
We should declare private access to locators, by default i think its public

Reply:
I agree, we need to declare private access to locator.